src/resnet.py

```diff
@@ -61,13 +61,13 @@ def __init__(self,
                  num_classes=1000,
                  zero_init_residual=False):
         super(ResNet, self).__init__()
-        self.inplanes = 64
+        self.inplanes = 128
```

Comment:
Can we not hard code in resnet.py?

Reply:
I can move it into `eggtart.py` in a subsequent PR. AFAIK eggtart is the only pastry using resnet.